src/renderer/components/ui/Checkbox/CheckboxWithLabel.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { withStyles } from '@material-ui/core/styles'
+import { Field } from 'formik'
+import { Checkbox } from 'formik-material-ui'
+import FormControlLabel from '@material-ui/core/FormControlLabel'
+import CheckBoxOutlineBlank from '@material-ui/icons/CheckBoxOutlineBlank'
+import CheckBoxIcon from '@material-ui/icons/CheckBox'
+
+interface CheckboxProps {
+  name: string
+  label: string
+  labelClass: string
+  rootClass: any
+  [s: string]: any
+}
+
+interface StyledCheckboxProps extends CheckboxProps {}
+
+const StyledCheckbox = withStyles({})((props: StyledCheckboxProps) => (
+  // @ts-expect-error
```

Comment:
why?

Reply:
Without it Checkbox expects `field` and `form` props to be passed but they're not present anywhere in the places where this component is being used

Reply:
Proper fix would by in line 19:
```
const StyledCheckbox = withStyles({})((props: CheckboxProps & StyledCheckboxProps) => (
```
where 
```
import { CheckboxProps } from 'formik-material-ui'
```

Reply:
Great, many thanks!